tfjs-core/src/ops/broadcast_to.ts

```diff
@@ -50,6 +51,8 @@ function broadcastTo_<R extends Rank>(
     throw new Error(`broadcastTo(): Invalid broadcast shape [${shape}].`);
   }
 
+  assertNonNegativeIntegerDimensions(shape);
```

Comment:
We can avoid having to check this for values we get from `convertToTensor` if we add this check to every `convertToTensor` call. [We already check dtype](https://github.com/tensorflow/tfjs/blob/master/tfjs-core/src/tensor_util_env.ts#L97-L100), so checking shape as well seems fine to me.

Reply:
Unfortunately, I don't think there's a similar fix for ops that make new tensors unless we make changes to the [`op` function](https://github.com/tensorflow/tfjs/blob/master/tfjs-core/src/ops/operation.ts#L27-L66), and even then, it's probably not really possible since you don't know the order of the args.